pyflakes/checker.py

```diff
@@ -1160,6 +1166,14 @@ def addBinding(self, node, value):
         # in scope
         if value.name not in self.scope or not isinstance(value, Annotation):
             self.scope[value.name] = value
+            scope = self.scope
```

Comment:
if I'm reading this correctly -- it's now assigning twice?

Reply:
@asottile You were... That assignment shouldn't have been there; I removed it.